packages/commonwealth/client/scripts/views/components/user_survey_popup.tsx

```diff
@@ -0,0 +1,156 @@
+/* @jsx m */
+
+import m from 'mithril';
+import app from 'state';
+
+import 'components/user_survey_popup.scss';
+
+import { CWButton } from './component_kit/cw_button';
+import { CWCheckbox } from './component_kit/cw_checkbox';
+import { CWGrowl } from './component_kit/cw_growl';
+import { CWText } from './component_kit/cw_text';
+import { CWIcon } from './component_kit/cw_icons/cw_icon';
+
+const USER_SURVEY_DISPLAY_INTERVAL = 1000 * 60 * 1; // How long to wait before redisplaying the survey
+
+type UserSurveyViewAttrs = {
+  disabled: boolean;
+  checked: boolean;
+  onRedirectClick: () => void;
+  onClose: () => void;
+  onCheckboxClick: () => void;
+};
+
+class UserSurveyView implements m.ClassComponent<UserSurveyViewAttrs> {
+  view(vnode) {
+    const { disabled, checked, onRedirectClick, onClose, onCheckboxClick } =
+      vnode.attrs;
+    return (
+      <CWGrowl position="bottom-right" disabled={disabled}>
+        <div class="UserSurveyPopup">
+          <div class="survey-svg-header"></div>
+          <CWIcon iconName="close" className="close-icon" onclick={onClose} />
+          <CWText type="h3" fontWeight="bold" className="header-text">
+            Want a Milk Carton NFT?
+          </CWText>
+          <CWText type="b1" className="body-text">
+            Take a quick survey to help us improve Common and get a special NFT
+            dropped to your ETH address!
+          </CWText>
+          <div class="button-wrapper">
```

Comment:
Let's make a CWButtonGroup to copy the syntax of Construct UI's Button > ButtonGroup structure (then we can also begin to replace those instances too maybe @affordances???)

Reply:
Did you mean in this PR or a different one?

Reply:
i meant in this PR, but that can be pushed back on. we should just make note to do it in the component kit